website/highlights/2020-06-29-batch-buffer-rework.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
I really like this change, but I don't think it's worth a highlight. This is expected behavior, imo. It should have always worked this way.